_Week8/Group4/results.do

```diff
@@ -0,0 +1,34 @@
+clear all
+
+global username "/Users/geenapanzitta/Documents/GitHub"
+
+cd "${username}/econ490-fall22/_Week8/Group4"
+
+run "simulate.do"
+
+set seed 5731341 //generated using random.org
+
+foreach j in 50 100 200 300 400 500 {
+	forv i = 1/100 {
+		clear
+		qui income_sim `j'
+		mat results = nullmat(results) \ [`i', `j', `r(b_age)', `r(b_educ_yrs)', `r(b_dist_from_city)', `r(b_family_size)', `r(b_experience)']
+	}
+}
+
+mat colnames results = i n b_age b_educ_yrs b_dist_from_city b_family_size b_experience
+
+clear
+
+svmat results, names(col)
+
+local vars b_age b_educ_yrs b_dist_from_city b_family_size b_experience
```

Comment:
Just flagging that if you apply my change on line 19, this line as well as line 31 will also have to be updated to reflect the new column names!

Reply:
I suggest to use a local macro to store the namelist so you don't have to apply parallel updates in situations like this!